workers/data_refinery_workers/processors/utils.py

```diff
@@ -0,0 +1,112 @@
+import os
+import shutil
+from django.utils import timezone
+from django.core.exceptions import ObjectDoesNotExist
+from typing import List, Dict, Callable
+from data_refinery_models.models import Batch, BatchStatuses, ProcessorJob
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# This path is within the Docker container.
+ROOT_URI = "/home/user/data_store/"
+
+
+def start_job(kwargs: Dict):
+    """Record in the database that this job is being started and
+    retrieve the job and batch from the database.
+    Retrieves the job and the job's batch from the database and
+    adds them to the dictionary passed in with the keys 'job'
+    and 'batch' respectively."""
+    job = kwargs["job"]
+    job.worker_id = "For now there's only one. For now..."
+    job.start_time = timezone.now()
+    job.save()
+
+    try:
+        batch = Batch.objects.get(id=job.batch_id)
+    except ObjectDoesNotExist:
```

Comment:
By the way, you can also write:
```
except Batch.DoesNotExist:
```
then you don't need to import `ObjectDoesNotExist` on line 4. Just a shortcut that I think is good to know.